src/components/App/App.tsx

```diff
@@ -48,6 +49,14 @@ const App = (): JSX.Element => {
     setUserSentiment(averageSentiment || newUserSentiment)
   }
 
+  const siftArticles = (selectedSections:string[]): void => {
+    const siftedArticles = articles.filter(article => {
+      return selectedSections.includes(article.section)
+    })
+    console.log(siftedArticles)
```

Comment:
Remember to remove console.logs :)

Reply:
Just FYI, there is another one in the handleChange function of the SectionForm component.